backend/run/copr-pulp-json-to-labels

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/python3
+
+"""
+In the past, it was not possible to add custom labels for Pulp content (RPM
+packages). For our purposes, we needed to label every package with a Copr build
+ID that produced it. As a temporary measure, we invented a `pulp.json`
+containing all Pulp hrefs associated with the build. When deleting a build from
+our storage, we know what objects to remove in Pulp.
+
+Pulp labels were implemented, and this script labels all existing results with
+appropriate build IDs.
+"""
+
+import os
+import json
+import logging
+import argparse
+from copr_common.tree import walk_limited
+from copr_common.log import setup_script_logger
+from copr_backend.helpers import BackendConfigReader
+from copr_backend.pulp import PulpClient
+
+
+log = logging.getLogger(__name__)
+setup_script_logger(log, "/var/log/copr-backend/pulp-json-to-labels.log")
+
+
+def get_arg_parser():
+    """
+    CLI argument parser
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--projects",
+        required=True,
+        help="Path to a file with a list of Pulp projects",
+    )
+    return parser
+
+
+def main():
+    """
+    The main function
+    """
+    parser = get_arg_parser()
+    args = parser.parse_args()
+    config = BackendConfigReader("/etc/copr/copr-be.conf").read()
+    pulp = PulpClient.create_from_config_file()
+
+    with open(args.projects, "r", encoding="utf-8") as fp:
+        projects = fp.read().split()
+
+    remove = []
+    for project in projects:
+        # TODO CoprDirs
```

Comment:
In our `/etc/lighttpd/pulp-redirect.txt` we have only `owner/project` names, not CoprDir names. So we will have to update this loop to go through all of them.